spec/computer_player_spec.rb

```diff
@@ -14,12 +14,6 @@ module TicTacToe
     let(:infinity) { Float::INFINITY }
     let(:neg_infinity) { -Float::INFINITY }
 
-    it 'implements methods required by Player interface' do
-      Player.required_methods.each do |method|
-        expect(ai).to respond_to(method)
-      end
-    end
-
     describe '#initialize' do
       it 'takes a parameters hash with the player\'s mark, opponent\'s mark, and the game board' do
```

Comment:
I would use double quotes for the `it` string. That way, you don't have to escape the single quotes.

In general, I like to favor using double quotes. There are a few main reasons. Keep in mind that String interpolation requires double quotes:

1. I don't have to change single quotes to double quotes if I want to start interpolating where I was not before.
2. I don't have a mixture of single quotes and double quotes. I like consistency.
3. Related to 2., I don't have to think about which quote to use.